installing/installing-fips.adoc

```diff
@@ -37,6 +37,9 @@ Because FIPS must be enabled before the operating system that your cluster uses
 |Use of FIPS compatible golang compiler.
 |TLS FIPS support is not complete but is planned for future {product-title} releases.
 
+|FIPS support across multiple architectures.
```

Comment:
Should we note `x86_64` in the intro as well for clarity (line 8)? Something like:
s/Starting with version 4.3, you can install an {product-title} cluster that uses FIPS Validated / Modules in Process cryptographic libraries./You can install an {product-title} cluster that uses FIPS Validated / Modules in Process cryptographic libraries on the `x86_64` architecture.
(^Off-topic but we should be able to also remove "Starting with version 4.3" now that OCP support only goes back to 4.5.)